pkg/vtpm/src/server.cpp

```diff
@@ -23,13 +26,18 @@ using namespace google::protobuf::io;
 #define CODED_STRM_HDR_LEN 4
 #define SERVER_PORT 8877
 #define LISTEN_BACKLOG_LIMIT 16
-#define MAX_ARGS 15
+#define MAX_ARGS 25
 #define BIN_PATH "/usr/bin/"
-#define CMD_OUTPUT_FILE "cmd.output"
+#define CLIENT_READ_TIMEOUT 2  //seconds
+#define failure -1
+#define success 0
+#define PATH_PREFIX clientWorkingDir + "/"
```

Comment:
Having a #define which looks like a constant but refers to a (local) variable is a really bad idea. Just inline the use of clientWorkingDir.

Reply:
Sorry about that. My intention was to make the code less verbose, and abbreviate this prefix part in a macro. I have changed it as per your suggestion.